google/cloud/storage/internal/read_object_range_request_test.cc

```diff
@@ -17,6 +17,16 @@
 
 using namespace storage::internal;
 
+namespace {
+HttpResponse CreateRangeRequestResponse(char const* content_range_header) {
```

Comment:
Similar to above:
s/content_range_header/content_range_header_value/

Reply:
Fixed.